src/analyze/analyze.c

```diff
@@ -1603,14 +1673,17 @@ static int dump_syscall_filters(int argc, char *argv[], void *userdata) {
                 if (k < 0) {
                         fputc('\n', stdout);
                         fflush(stdout);
-                        log_notice_errno(k, "# Not showing unlisted system calls, couldn't retrieve kernel system call list: %m");
+                        log_notice_errno(
+                                k,
+                                "# Not showing unlisted system calls, couldn't retrieve kernel system call list: %m");
                 } else if (!set_isempty(kernel)) {
                         const char *syscall;
                         Iterator j;
 
                         printf("\n"
                                "# %sUnlisted System Calls%s (supported by the local kernel, but not included in any of the groups listed above):\n",
-                               ansi_highlight(), ansi_normal());
+                               ansi_highlight(),
+                               ansi_normal());
```

Comment:
hmm, i would not treat these arguments any other way, the formatter does not know about relevance of arguments, of the combination of "ansi_highlight(), ansi_normal()" is trivial then it would be probably better to write a function which does this by default and use this instead.